libraries/FunctionalInterrupt/src/FunctionalInterrupt.cpp

```diff
@@ -0,0 +1,23 @@
+#include "FunctionalInterrupt.h"
```

Comment:
The current filename is misleading, it confused me for a moment. Please change it to ScheduledInterrupts or similar.

Reply:
I will happily do so, I feel the same way, just didn't dare do it by myself.